content/en/developers/metrics/_index.md

```diff
@@ -44,7 +53,7 @@ Metrics reported by the Agent are in a pseudo-hierarchical dotted format (e.g. `
 
 **Note**: Metric names are case sensitive in Datadog.
```

Comment:
Move this further up since it causes pain for customers
This section should be edited to: 
"The following custom metric naming convention should be followed: 
* Metric names must start with a letter 
* Metrics name must only contain ASCII alphanumerics, underscores and periods. Unicode is _not_supported
* Other characters, including spaces, will be converted to underscores
* Metric names cannot exceed 200 characters. Fewer than 100 characters should be used for an optimal UI experience.

Reply:
I'm really not sure what value /question this answers from a customer perspective? 

"Metrics reported by the Agent are in a pseudo-hierarchical dotted format (e.g. `http.nginx.response_time`). The hierarchy is neither enforced nor interpreted, but it can be used to infer things about servers. For example, if `hostA` and `hostB` are both reporting `http.nginx.*` those must be web frontends."

Reply:
there is not let's remove this indeed.